metastore/src/main/scala/com/gu/tableversions/metastore/VersionPaths.scala

```diff
@@ -8,6 +8,7 @@ import com.gu.tableversions.core.{Partition, Version}
   * Encodes the mapping between version numbers and storage paths.
   */
 object VersionPaths {
+  val KEY_PREFIX = "versioned__"
```

Comment:
Wouldn't it be more standard practice to prefix config keys with something like `com.gu.tableversions` or similar?

Reply:
I stuffed it (for now at least) into VersionedFileSystem's companion object.